pkg/reporting/config.go

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright 2019 Chef Software, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package reporting
+
+import (
+	"github.com/chef/chef-analyze/pkg/config"
+	"github.com/chef/chef-analyze/pkg/credentials"
+)
+
+type Config struct {
```

Comment:
Talking with @tyler-ball we decided to call this struct `Reporting` to avoid confusion with the actual `config.Config` cc/ @marcparadise